api/src/app.ts

```diff
@@ -128,7 +128,7 @@ try {
  * @param {NextFunction} next
  */
 function validateAllResponses(req: Request, res: Response, next: NextFunction) {
-  const isStrictValidation = !!req['apiDoc']['x-express-openapi-validation-strict'] || false;
+  const isStrictValidation = !!req['apiDoc']?.['x-express-openapi-validation-strict'] || false;
```

Comment:
nit: probably not needed since there are no other api changes

Reply:
Fixed